esp-hal/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- RMT fix filter threshold, value check no longer required (#3192)
```

Comment:
Reword this to say the change in the limit, 127 vs 255

Reply:
Is this OK
RMT fix filter threshold, change in limit was 127 now is 255 (#3192)

Reply:
RMT now uses correct max filter threshold of 255 instead of 127

Reply:
Sounds good